docs/source/modin/resampling.rst

```diff
@@ -2,7 +2,7 @@
 Resampling
 =============================
 
-.. currentmodule:: snowflake.snowpark.modin.pandas.resample
+.. currentmodule:: modin.pandas
```

Comment:
We still specify `snowflake.snowpark.modin.plugin.extensions.groupby_overides` for GroupBy, can you check if the documentation build here is still using Snowpark pandas docs?

Reply:
I switched it to `snowflake.snowpark.modin.plugin.extensions.resample_overrides`